components/suggest/src/rs.rs

```diff
@@ -266,3 +268,22 @@ pub(crate) struct DownloadedPocketSuggestion {
     pub high_confidence_keywords: Vec<String>,
     pub score: f64,
 }
+
+/// A Yelp dataset to ingest from a Yelp Attachment
+#[derive(Clone, Debug, Deserialize)]
+pub(crate) struct LocationSign {
+    pub keyword: String,
+    pub need_location: i8,
```

Comment:
Is there a reason this can't be a `bool`? (I don't know anything about Serde yet so I'm happy to learn 🙂)

Reply:
Oh, indeed, it seems that we can use `bool` here. Thanks!